client/src/lib/slate-pointers/exportedPointerSpacer.ts

```diff
@@ -0,0 +1 @@
+export const SPACER = "\u202f"; // non-breaking thin space
```

Comment:
I'm not a big fan of the word SPACER. Given how important it is, I'd prefer something descriptive to something short. Maybe something like, "POINTER_EDGE_SPACE"?